Lib/fontbakery/specifications/hhea.py

```diff
@@ -47,10 +47,15 @@ def com_google_fonts_check_073(ttFont):
   id = 'com.google.fonts/check/079',
   conditions = ['glyph_metrics_stats']
 )
-def com_google_fonts_check_079(ttFont):
+def com_google_fonts_check_079(ttFont, glyph_metrics_stats):
   """Monospace font has hhea.advanceWidthMax equal to each glyph's
   advanceWidth?"""
 
+  seems_monospaced = glyph_metrics_stats["seems_monospaced"]
+  if not seems_monospaced:
+    yield PASS, ("Font is not monospaced.")
+    return
```

Comment:
Maybe it works to simply `return SKIP, msg` instead of yielding and then returning.

Reply:
Nope... if I do that, I get:
```
 * ERROR: The check <FontBakeryCheck:com.google.fonts/check/079> did not yield any status

   Result: ERROR
```